documentation/Installing-Clear-Containers-on-Centos-7.md

```diff
@@ -1,11 +1,17 @@
 # Install the Clear Containers runtime on Centos 7
 
-
 ## Introduction
 
-Installing Clear Containers on CentOS is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of gcc and glibc.
+Installing Clear Containers on CentOS is slightly more difficult than on other
+distributions mainly due to the fact that it does not provide current versions
+of gcc and glibc.
+
+Hence, this guide explains how to install updated packages in `/usr/local`,
+to avoid conflicting with the base packages.
 
-Hence, this guide explains how to install updated packages in `/usr/local`, to avoid conflicting with the base packages.
+Clear containers supports latest version of Docker CE (currently 17.05), with
+the exception of Swarm. If you want to use Swarm, you should install Docker
+version 1.12.1.
```

Comment:
@chavafg I think the dns resolution was broken in 1.12.1 as well, we just did not have a test for it. Could you verify.